continuousintegration.py

```diff
@@ -24,7 +24,9 @@ def watch_ci():
     s.listen(10)
     print 'listening for CI changes'
 
-    while 1:
+    api_url = 'https://api.github.com/repos/Charcoal-SE/SmokeDetector/'
```

Comment:
I refactored it a little to use the named parameters, see if you like that, otherwise I'll return it